### PR TITLE
SAM-2995 wrap question text in a label

### PR DESCRIPTION
**Pull request by ottenhoff**
rA11y

### CODE REVIEW

File: samigo/samigo-app/src/webapp/jsf/delivery/item/deliverCalculatedQuestion.jsp [hunk 1]
```diff
@@ -27,10 +27,10 @@ should be included in file importing DeliveryMessages
   <!-- ATTACHMENTS -->
 <%@ include file="/jsf/delivery/item/attachment.jsp" %>
 
-<samigo:dataLine value="#{question.finArray}" var="answer"
-  separator=" " first="0" rows="100">
+<samigo:dataLine value="#{question.finArray}" var="answer" separator=" " first="0" rows="100">
   <h:column>
-      <h:outputText value="#{answer.text} " escape="false" />
+      <h:outputLabel for="calcq" value="#{answer.text} " escape="false" />
+      <f:verbatim>&nbsp;</f:verbatim>
```
Comment (master-bob) [inline samigo/samigo-app/src/webapp/jsf/delivery/item/deliverCalculatedQuestion.jsp]: What's the benefit of this space?

Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/delivery/item/deliverCalculatedQuestion.jsp]: not having an input text box right up against the label text

Comment (master-bob) [inline samigo/samigo-app/src/webapp/jsf/delivery/item/deliverCalculatedQuestion.jsp]: Isn't that what CSS is for?

Comment (ottenhoff) [inline samigo/samigo-app/src/webapp/jsf/delivery/item/deliverCalculatedQuestion.jsp]: Sure, ideally. But I'm looking to make a number of a11y changes quickly without spending a ton of time on each of these.

**Pull request merged**